applications/CoSimulationApplication/custom_data_structure/pyKratos/Matrix.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import print_function, absolute_import, division
+class Element:
```

Comment:
Why `Matrix.py` implements the same as `Element.py`?

Reply:
removed